scripts/calculator.js

```diff
@@ -90,11 +79,15 @@ const clickListenerSubtract = () => {
 const clickListenerAdd = () => {
   updateScreen('+')
 }
+
+/* 
+  Update this so that only one decimal can show up in either number.
+  It will look similar to 'addNumber' with some additional logic.
+*/
 const clickListenerDecimal = () => {
```

Comment:
This will be trickier.

If you were to use my `addNumber` function here you could wind up with numbers like "4353..3453...353."

You'll need to create a modified version of the function that will only allow one decimal to be used.